index.html

```diff
@@ -796,11 +802,11 @@
        starts the domain name lookup, record the time as <a for="PerformanceResourceTiming">domainLookupStart</a>.</li>
       <li>Record the time as <a for="PerformanceResourceTiming">domainLookupEnd</a> immediately after the
        domain name lookup is successfully done. A user agent may need multiple retries before that. If
-       the domain lookup fails, abort the remaining steps. </li>
+                the domain lookup fails, record the time immediately before the user agent aborts the same domain or Timing Allow Origin enabled request for resources, otherwise abort the remaining steps.</li>

```

Comment:
Perhaps... 

"Record the time as _domainLookupEnd_ immediately after the domain name lookup is successfully done, or if the lookup fails and resource passes the _timing allow check_. A user agent may need multiple retries before that. If the domain lookup fails, go to step X."